x-pack/packages/security/api_key_management/src/components/api_key_flyout.tsx

```diff
@@ -237,6 +237,7 @@ export const ApiKeyFlyout: FunctionComponent<ApiKeyFlyoutProps> = ({
         (accumulator, roleName) => {
           const role = roles.find((r) => r.name === roleName);
           if (role) {
+            // @ts-expect-error
```

Comment:
@JoshMock

Reply:
This one is more about our code assuming it's a type when the type coming from ES can also be other types of role.

Reply:
[PR in progress](https://github.com/elastic/kibana/pull/205174)